v5/cli/conductor.rst

```diff
@@ -1,3 +1,79 @@
-===============
-Conductor (CLI)
-===============
+====================
+Conductor User Guide
+====================
+
+Introduction
+============
+
+Conductor is PROS's templating and project management facility. It is
+responsible for resolving versions of the PROS kernel, user libraries. It also

```

Comment:
`the PROS kernel and user libraries`, unless there is something else you want to list here.